pkg/xen-tools/patches-4.19.0/11-char-socket-revert.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/tools/qemu-xen/chardev/char-socket.c b/tools/qemu-xen/chardev/char-socket.c
+index 8c58532..e816841 100644
+--- a/tools/qemu-xen/chardev/char-socket.c
++++ b/tools/qemu-xen/chardev/char-socket.c
+@@ -125,8 +125,8 @@ static int tcp_chr_write(Chardev *chr, const uint8_t *buf, int len)
+ 
+         if (ret < 0 && errno != EAGAIN) {
+             if (tcp_chr_read_poll(chr) <= 0) {
+-                /* Perform disconnect and return error. */
+                 tcp_chr_disconnect_locked(chr);
++               return len;
```

Comment:
If you don't want to ignore the error, then you don't need this patch at all... because the function will return `ret` with the appropriate error code.... `return len` just makes the function tell to the caller that the whole buffer was filled...